build_schema.py

```diff
@@ -106,7 +111,41 @@ def loadYAML(location):
         if "x-kubernetes-group-version-kind" in type_def:
             for kube_ext in type_def["x-kubernetes-group-version-kind"]:
                 if "properties" not in type_def:
+                    invalid_specs_groups.append(type_def["x-kubernetes-group-version-kind"][0]["group"])
+                    invalid_specs_kinds.append(type_def["x-kubernetes-group-version-kind"][0]["kind"])
                     delete_list.append(type_name)
+                else:
+                    valid_specs_groups.append(type_def["x-kubernetes-group-version-kind"][0]["group"])
+                    valid_specs_kinds.append(type_def["x-kubernetes-group-version-kind"][0]["kind"])
+
+    try:
+        with open(invalid_kinds_fp, "r+") as skf:
+            skip_kinds_file = skf.read()
+            skip_kinds_file = skip_kinds_file.rstrip('\n')
+            skip_kinds_file = skip_kinds_file.split(",")
```

Comment:
can be replaced with
```suggestion
            skip_kinds_file =  skf.readlines()[0].split(",")
```